lib/util/exec/hermit.spec.ts

```diff
@@ -0,0 +1,88 @@
+jest.mock('../../config/global');
+jest.mock('fs-extra');
+jest.mock('./common');
+
+import fs from 'fs-extra';
+import { mocked, mockedFunction } from '../../../test/util';
+import { GlobalConfig } from '../../config/global';
+import { rawExec } from './common';
+import { findHermitCwd, getHermitEnvs, isHermit } from './hermit';
+import type { ExecResult, RawExecOptions } from './types';
+
+const globalConfigMock = mocked(GlobalConfig);
+const fsMock = mocked(fs);
+const rawExecMock = mockedFunction(rawExec);
+
+describe('util/exec/hermit', () => {
+  describe('isHermit', () => {
+    it('should return true when binarySource is hermit', () => {
+      globalConfigMock.get.mockReturnValue({ binarySource: 'docker' });
```

Comment:
if possible, we prefer to use `jest.spyOn` for mocking,
please see https://github.com/renovatebot/renovate/blob/main/docs/development/best-practices.md#unit-testing